assemblyline/odm/models/config.py

```diff
@@ -520,7 +535,7 @@ class Scaler(odm.Model):
                                                            description="Defaults Scaler will assign to a service.")
     cpu_overallocation: float = odm.Float(description="Percentage of CPU overallocation")
     memory_overallocation: float = odm.Float(description="Percentage of RAM overallocation")
-    overallocation_node_limit = odm.optional(odm.integer(description="If the system has this many nodes or "
+    overallocation_node_limit = odm.Optional(odm.Integer(description="If the system has this many nodes or "
```

Comment:
just FYI the lower case versions of the odm objects are functions with type casting so that mypy or other type checkers can figure out the types of the model fields without annotation